docs/release-notes/dictionary-releases.md

```diff
@@ -19,6 +19,38 @@ Coming soon.
 None to report.
 ------>
 
+## Release 1.8
+
+**Release Date:** April 27, 2021
+
+### Updates
+Two **new** data type tables have been added to the dictionary for submission:
```

Comment:
could say `clinical files` instead of `data type tables` as that's what the table of contents header is?

Reply:
could also say "...have been added to the dictionary and the file template downloads for submission"